orchagent/routeorch.cpp

```diff
@@ -673,6 +761,10 @@ void RouteOrch::doTask(Consumer& consumer)
                     {
                         ctx.protocol = fvValue(i);
                     }
+                    if (fvField(i) == "fallback_to_default_route")
```

Comment:
who sets this attribute?

Reply:
it is being done here: https://github.com/sonic-net/sonic-swss/pull/3353